prow/config.yaml

```diff
@@ -1,5 +1,26 @@
 plank:
-  job_url_template: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/{{if eq .Spec.Type "presubmit"}}pr-logs/pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}logs{{end}}{{if ne .Spec.Refs.Org ""}}{{if ne .Spec.Refs.Org "kubernetes"}}/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}{{else if ne .Spec.Refs.Repo "kubernetes"}}/{{.Spec.Refs.Repo}}{{end}}{{end}}{{if eq .Spec.Type "presubmit"}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if eq .Spec.Type "batch"}}/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
+  job_url_template: >-
+    https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/
+      {{- if eq .Spec.Type "presubmit" -}}
```

Comment:
I was under the impression that this puts spaces in place of newlines. Is that not right?

Reply:
Hmm -- the one unit test verifying this seems to be OK with it. I'll make a playground

Reply:
Ah sorry -- a little bit of trickery.. this is a YAML folded block so there are no newlines in the string

http://www.yaml.org/spec/1.2/spec.html#id2796251